lib/membrane_audio_mixer/clip_preventer.ex

```diff
@@ -0,0 +1,92 @@
+defmodule Membrane.AudioMixer.ClipPreventer do
```

Comment:
I came up with a new proposal for the names - `Adder` and `ClipPreventingAdder`. Lemme know what 'ya think 'bout that :wink: